tests/test_create_run.py

```diff
@@ -75,19 +75,3 @@ def test_010_drop_secondary():
     node1.drop()
     assert not node1.pg_is_running()
     assert node2.wait_until_state(target_state="single")
-    
-def test_011_add_new_secondary():
-    global node3
-    node3 = cluster.create_datanode("/tmp/basic/node3")
-    node3.create(run = True)
-    assert node3.wait_until_state(target_state="secondary")
-    assert node2.wait_until_state(target_state="primary")
-
-def test_012_multiple_manual_failover():
```

Comment:
any reason why we don't need this anymore ?

Reply:
I don't believe those tests are expecting any specific behaviour in `create --run` and we test them already in the normal `create` then `run` cases.